src/components/Icon.stories.tsx

```diff
@@ -48,6 +48,7 @@ export const Icon = (props: IconProps) => {
     "duplicate",
     "filter",
     "columns",
+    "bxcolumns"
```

Comment:
**question:** Do you know if design had a specific reason to use a second "columns" icon? Having two icons to represent the same thing within the same application seems confusing to me. I wouldn't be surprised if the designer was just unaware of the other icon. Or if they are aware, I'd want to double check that we really need a second `columns` icon.

Reply:
@aeppy

Reply:
@bdow Sure thing, will do!

Reply:
I'm also okay with us naming it something else. Looking at the Figma file, it appears we named it `kanban`, which isn't as confusing. One icon for "columns" and on for "kanban" helps me understand their purpose a bit better. This PR had them both named "columns" (granted, one was "bxcolumns"), which is why this was red flag to me. But if they do have different purposes, and are named for those different purposes, then I'm good with that too.

Reply:
Great, good to know. We may not need 2 anyway. I just asked the design team and lauren said she removed the columns icon where she was using it in schedules, and I don't think we use it elsewhere. But I'm waiting to hear back from the rest of the team just to be sure.

Reply:
Awesome! I just did a quick check in Blueprint, and it does not appear that we use this icon anywhere at the moment.

Reply:
@bdow should I replace columns icon with this one?

Reply:
That will be up to design. I'd wait for their response.

Reply:
Just waiting to hear back from the rest of the design team if they're cool with replacing it. Only heard from Lauren so far. But yes, that's the plan if everyone agrees.

Reply:
Ok no one else chimed in so I will assume they're ok with it since I believe that's the only place in blueprint it's used. We can go ahead and replace the icon, @JonnCharpentier.

Reply:
Awesome! Thanks @aeppy for asking the team about. Can we update the Beam Figma to remove the old Column icon and change the name of the new one from `kanban` to `columns`?

Reply:
Or if we want to keep the name as `kanban`, I'm okay with that too. Just want to make sure the naming is consistent between Figma and the implementation.

Reply:
Yep! Will do! I'll keep it as kanban. @JonnCharpentier @bdow